plugins/source/aws/docs/tables/aws_glue_registry_schema_versions.md

```diff
@@ -14,16 +14,16 @@ This table depends on [aws_glue_registry_schemas](aws_glue_registry_schemas.md).
 
 | Name          | Type          |
 | ------------- | ------------- |
-|_cq_id (PK)|`uuid`|
+|_cq_id|`uuid`|
 |_cq_parent_id|`uuid`|
 |account_id|`utf8`|
 |region|`utf8`|
-|registry_schema_arn|`utf8`|
+|registry_schema_arn (PK)|`utf8`|
 |metadata|`json`|
 |created_time|`utf8`|
 |data_format|`utf8`|
 |schema_arn|`utf8`|
 |schema_definition|`utf8`|
-|schema_version_id|`utf8`|
+|schema_version_id (PK)|`utf8`|
 |status|`utf8`|
 |version_number|`int64`|
```

Comment:
Why include `schema_version_id` and not `version_number`?

Reply:
In the [`GetSchemaVersion`](https://docs.aws.amazon.com/glue/latest/webapi/API_GetSchemaVersion.html) API call either schema ID or schema version ID is required. Number can be `latest`, so the `version_id` seems a more suitable candidate to be included in the PK.